TFNetwork.py

```diff
@@ -1967,6 +1972,57 @@ def copy_new_base(self, name=None, layer=None, network=None, reduce_func=None):
       norm_factor=self._norm_factor, only_on_eval=self._only_on_eval)
 
 
+class MetaLossHolder:
```

Comment:
It is not a good idea to introduce a class which is sort-of-like `LossHolder` but not quite. It should properly derive from the base class. You can throw an exceptions in any functions which are not supported or so.